gslib/tests/test_rpo.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for rpo command."""
```

Comment:
All caps "RPO"

Reply:
I would prefer calling it `rpo` because that's the command name.

Reply:
Ahhh, sounds good